tests/page-objects/default/reports/reports.wdio.page.js

```diff
@@ -205,51 +220,44 @@ const isReportSelected = async (uuid) => {
   return await (await checkbox).isExisting();
 };
 
-const selectReports = async (uuids) => {
+const toggleSelectReport = async (selectReport, uuids, verifyFn, shouldHideElements = false) => {
```

Comment:
You can remove the 'verifyFn' argument, since 'verifyMultiselectElementsDisplay' is the only function being used.

```suggestion
const toggleReportSelection = async (uuids, shouldSelect, shouldHideElements = false) => {
```

Reply:
I pass the function as parameter because I am using the same function in the reports page for `default-mobile`, [here](https://github.com/medic/cht-core/blob/8704-clean-up-and-organize-reports-folder/tests/page-objects/default-mobile/reports/reports.wdio.page.js#L36-L44), and the method `verifyMultiselectElementsDisplay` is different for `default` and `default-mobile`

Reply:
My bad, sorry. Let's simplify this - we're making everything worse. What do you think about leaving as it was?

Reply:
Yes, that was my exact same feeling, that maybe it was getting more complicated than it should
I will change it back.
Thanks Lore!